disperser/apiserver/server.go

```diff
@@ -255,6 +256,13 @@ func (s *DispersalServer) disperseBlob(ctx context.Context, blob *core.Blob, aut
 		}
 	}
 
+	// validate every 32 bytes is a valid field element
+	_, err = rs.ToFrArray(blob.Data)
```

Comment:
It's better to make this validation early so we can reject it early if it's invalid

Reply:
moved to "validateRequestAndGetBlob" after rate limit, @mooselumph